src/mango_idx_text.erl

```diff
@@ -226,6 +227,9 @@ construct_analyzer({Props}) ->
     end.
 
 
+%% Empty Selector
+indexable_fields({[]}) ->
+    [];
```

Comment:
I'd like to keep this for situations where the text search is uses a {} selector.